### PR TITLE
LPS-183677 Creation of task fixed, through the correct taskid

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
Ci:test:sf

**Review by EugenioAntunes — APPROVED** [review]
Nice job, Man 🐐

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/hoc/withPagePermission.tsx [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
 import {Component, useMemo} from 'react';
-import {Navigate, useOutletContext} from 'react-router-dom';
+import {Navigate, useMatch, useOutletContext} from 'react-router-dom';
 import Loading from '~/components/Loading/Loading';
 import Rest from '~/core/Rest';
 import {useFetch} from '~/hooks/useFetch';
 import {ObjectActionsItems} from '~/services/rest';
 
 type PageProperties = {
+	createPath: string;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/hoc/withPagePermission.tsx:23]:
```suggestion
	createPath?: string;
```

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/hoc/withPagePermission.tsx [hunk 2]
```diff
@@ -31,10 +32,12 @@ type CheckPermissionProps = {
 
 const CheckPermission: React.FC<CheckPermissionProps> = ({
 	children,
-	properties: {restImpl, redirectTo = `/404?type=permission`},
+	properties: {createPath, restImpl, redirectTo = `/404?type=permission`},
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/hoc/withPagePermission.tsx:35]:
```suggestion
	properties: {createPath = '', restImpl, redirectTo = `/404?type=permission`},
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Review by Elvinho — APPROVED** [review]
Nice 🔥

**Review by kevenleone — APPROVED** [review]
LGTM 🔥

**Comment by kevenleone** [comment]
ci:forward

**Pull request closed without merging**